chichiwang/chichiwang.dev
# Remove boilerplate `app/Layout`

## Description
Linked to Issue: #3
Remove boilerplate layout. It is no longer used. It has been replaced with a new baseline site layout in PR #14 

## Changes
* Remove directory `app/Layout`

## Steps to QA
* Pull down and switch to this branch
* Run the app and ensure no issues
* Verify in browser no errors on any of the application routes (`/`, `/error`, `/notfound`)
